backend/src/hatchling/builders/constants.py

```diff
@@ -30,3 +30,6 @@ class BuildEnvVars:
     HOOK_ENABLE_PREFIX = 'HATCH_BUILD_HOOK_ENABLE_'
     CLEAN = 'HATCH_BUILD_CLEAN'
     CLEAN_HOOKS_AFTER = 'HATCH_BUILD_CLEAN_HOOKS_AFTER'
+
+
+EDITABLES_MINIMUM_VERSION = '0.3'
```

Comment:
Can we please switch this to `EDITABLES_REQUIREMENT` with value `editables ~= 0.3` so that we can just reference directly everywhere?

Reply:
https://github.com/pypa/hatch/pull/1255/commits/76272545f2db474c79708bb05a2a8bb0d19680fb